Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby "2.2.3"
+ruby "2.3.0"
```

Comment:
Sounds great, but i don't see that as a blocker to getting this landed. Seems like it's another PR.